doc/stdenv/platform-support-tiers.chapter.md

```diff
@@ -0,0 +1,88 @@
+# Platform Support Tiers {#platform-support-tiers}
```

Comment:
```suggestion
# Non-normative Platform Support Tiers {#platform-support-tiers}
```

The RFC specifies these as "non-normative" (and also nearly *four years out of date* at this point).